packages/services/storage/src/index.ts

```diff
@@ -423,6 +423,8 @@ export async function createStorage(
       | 'validation_percentage'
       | 'validation_period'
       | 'validation_excluded_clients'
+      | 'validation_request_count'
```

Comment:
I still need to generate the `targets` type using `schemats`. Working on the migration now.